db/models.py

```diff
@@ -50,3 +54,51 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE)
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey("MovieSession", on_delete=models.CASCADE)
+    order = models.ForeignKey("Order", on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    def __str__(self) -> str:
+        return f"{self.movie_session} (row: {self.row}, seat: {self.seat})"
+
+    def clean(self) -> None:
+        if self.row > self.movie_session.cinema_hall.rows:
+            raise ValidationError(
+                {"row":
+                    f"row number must be in available range: "
+                    f" (1, rows): (1, {self.movie_session.cinema_hall.rows})"
+                 })
+
+        if self.seat > self.movie_session.cinema_hall.seats_in_row:
+            raise ValidationError(
+                {"seat":
+                    "seat number must be in available range: "
+                    f"(1, seats_in_row): (1, "
+                    f"{self.movie_session.cinema_hall.seats_in_row})"
+                 })
```

Comment:
In the `clean` method, ensure that the `seat` number is a positive integer. Currently, it only checks if the seat is greater than the available seats in a row, but it should also check if it's a positive number.